test/scenarios.py

```diff
@@ -4247,57 +4179,48 @@
                             event.Progress(0, FILES["nested/big/testfile-02"].id),
                         ]
                     ),
-                    action.CancelTransferRequest(0),
-                    action.Wait(event.FinishTransferCanceled(0, False)),
-                    # new transfer
-                    action.Wait(
-                        event.Receive(
-                            1,
-                            "172.20.0.5",
-                            {
-                                event.File(
-                                    FILES["nested/big/testfile-01"].id,
-                                    "nested/big/testfile-01",
-                                    10485760,
-                                ),
-                                event.File(
-                                    FILES["nested/big/testfile-02"].id,
-                                    "nested/big/testfile-02",
-                                    10485760,
-                                ),
-                            },
-                        )
+                    action.CancelTransferFile(0, FILES["nested/big/testfile-01"].id),
+                    action.CancelTransferFile(0, FILES["nested/big/testfile-02"].id),
```

Comment:
This whole testsuite is different mainly because it actually cancels the file instead of the transfer request as it did previously, what is the reason of not leaving the old test and making a new one?

Reply:
The old test exercised the resume across the transfer. The new exercises resume within a single transfer.
With per transfer temporaries, by definition, the resume across the transfers would not work

Reply:
It makes sense, canceling a transfer is basically finalizing it and no resume should happen there, yay!